enferno/utils/backup_utils.py

```
@@ -13,20 +12,22 @@
 def pg_dump(filepath):
     # localhost with no password set
     if Config.get("POSTGRES_HOST") == "localhost" and not Config.get("POSTGRES_PASSWORD"):
-        cmd = ["pg_dump", "-Fc", f"{Config.get("POSTGRES_DB")}", "-f", f"{filepath}"]
+        cmd = ["pg_dump", "-Fc", f"{Config.get('POSTGRES_DB')}", "-f", f"{filepath}"]
         return check_output(cmd)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)